.github/workflows/codeceptjs.yml

```diff
@@ -4,12 +4,12 @@ on: [pull_request]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
```

Comment:
So, I selfishly would like to keep these tests running properly on Ubuntu latest, otherwise they will also start failing for devs using linux laptops. I also wouldn't be surprised if the same chromium changes come to mac soon anyhow. Maybe we can simplify or drop the copy paste part of those tests? It seems like we are trying to test browser functionality more or less anyhow. If the input type is set, it's the browser controlling copy operations.

Reply:
We already used mac for tests and liniting. Worked fine. Changed it to ubuntu now.
I agree that we could meet problems when launching it on ubuntu locally. However I also think that chrome support is better on mac and windows as they are just more widespread.
I think it makes sense to remove the copy / paste part of the tests and leave it up to the browser used.